src/xcode.rs

```diff
@@ -22,13 +22,19 @@ pub(crate) fn integ(meta: &Meta, mut invocation: Invocation) -> Result<()> {
     Ok(())
 }
 
-fn targets_from_env() -> Result<Vec<&'static str>> {
+fn targets_from_env() -> Result<Vec<String>> {
     let archs = env::var("ARCHS").with_context(|e| format!("Failed to read $ARCHS: {}", e))?;
+    let platform_name = env::var("PLATFORM_NAME").with_context(|e| format!("Failed to read PLATFORM_NAME: {}", e))?;
```

Comment:
Do you know if this will break on earlier versions of Xcode? 

Maybe, to be on the safe side, can you update this code so that, if `PLATFORM_NAME` is not set, it defaults to the old behavior?

Reply:
I can't find any official documentation about when Xcode starts to set `PLATFORM_NAME`.
Defaulting to the old behavior surely is more reasonable. Updated as suggested.